sdk/keyvault/azure-security-keyvault-keys/src/private/key_serializers.hpp

```diff
@@ -97,11 +97,11 @@ namespace Azure { namespace Security { namespace KeyVault { namespace Keys { nam
   };
 
   /**************** Key Properties ************/
-  struct KeyPropertiesPageResultSerializer final
+  struct KeyPropertiesPagedResultSerializer final
```

Comment:
Haha, good thing this is details.

Paged**Response**, not Result :)

Same elsewhere, if we wanted to change it.